src/components/theme/View/FileView.jsx

```diff
@@ -25,7 +25,7 @@ const FileView = ({ content }) => (
       <p className="documentDescription">{content.description}</p>
     )}
     {content.file?.download && (
-      <a href={flattenToAppURL(content.file.download)}>
+      <a href={addPrefixPath(flattenToAppURL(content.file.download))}>
```

Comment:
do not use addPrefixPath.. use UniversalLink instead <a> tag, it uses router that makes the magic with prefix path!

Reply:
Changed